lib/acme_server/db.ex

```diff
@@ -1,24 +1,36 @@
 defmodule AcmeServer.Db do
   use GenServer
 
+  @type site :: String.t()
+  @type dns :: %{String.t() => String.t()}
+  @type config :: %{site: site, site_uri: URI.t(), dns: dns}
```

Comment:
Same thing here. These types are duplicates of the ones defined `AcmeServer`, so they are not needed.